src/tree.component.ts

```diff
@@ -83,4 +86,16 @@ export class TreeComponent implements OnInit, OnChanges {
       this.nodeCollapsed.emit(e);
     });
   }
+
+  public getController(): TreeController {
+    return this.rootComponent.controller;
+  }
+
+  public getChildControllerById(id: number | string): TreeController {
```

Comment:
You're not able to get a controller of a node from an other tree. It doesn't mater how many trees you have in your component or app, you will be able to get a controller only for a node inside the tree.
It is possible to get the root node by getChildControllerById. I think this could mislead users. It is possible to remove the option to get the root controller from getChildControllerById. What do you think?